src/Pages/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,20 @@
+import "./Dashboard.css"
+
+const Dashboard = () => {
+    return (
+        //Add Nav component
+        //Add SideMenu component
+        <section className="dashboard-page">
```

Comment:
[OPTIONAL] Nice work and understanding! If it is possible, ask the design team and follow the Figma design.